solr/core/src/test/org/apache/solr/handler/component/StatsComponentTest.java

```diff
@@ -1248,17 +1248,16 @@ public void testMiscQueryStats() throws Exception {
 
     // force constant score for matches, so we aren't dependent on similarity
     final float constScore = 4.2F;
-    final double expectedScore = (double) constScore;
```

Comment:
subjective: clearer before IMO i.e. fewer casts and clear that it's the expected score

Reply:
makes sense...